packages/bpk-docs/src/components/DocsPageBuilder/ComponentVideos.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import BpkText from 'bpk-component-text';
+import BpkPanel from 'bpk-component-panel';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './ComponentVideos.scss';
+
+const getClassName = cssModules(STYLES);
+
+let counter = 0;
+const generateId = (title = '') => {
+  counter += 1;
+  return `${title.toLowerCase().replace(' ', '-')}-${counter}`;
+};
+
+const ComponentVideo = ({ title, subText, src, ...rest }) => {
+  const subTextId = generateId(title);
+
+  return (
+    <div key={title} className={getClassName('bpkdocs-component-videos__item')}>
+      <dt className={getClassName('bpkdocs-component-videos__item-title')}>
+        <BpkText tagName="h3">{title}</BpkText>
+      </dt>
+      <dd
+        className={getClassName(
+          'bpkdocs-component-videos__item-video-container',
+        )}
+      >
+        <BpkPanel>
+          <video
+            loop
+            controls
+            className={getClassName('bpkdocs-component-videos__item-video')}
+            autoPlay
+            {...rest}
+          >
+            <track kind="captions" />
```

Comment:
Why are we including subtitles? It makes this appear in Safari (see screenshot) which I imagine we don't want.

<img width="280" alt="screen shot 2018-07-04 at 15 00 30" src="https://user-images.githubusercontent.com/73652/42281301-0d477ab8-7f9b-11e8-86e3-3712712d29b7.png">

Reply:
eslint fails if no track is provided, so we either blacklist the rule or add an empty caption that we might fill in later for accessibility purposes, preferences?

Reply:
Can we not just add a caption? :) Describe what is being shown in the video?

Reply:
If the video has the `muted` prop, the rule shouldn't fail: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/media-has-caption.md

However, I just tried this and it does still fail for me. This may be because we have an older version as this was a recent fix: https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/357

So I say opt out of the rule for now and when we upgrade eslint (which I believe is very soon) we can revisit this.

Reply:
> Can we not just add a caption? :) Describe what is being shown in the video?

We could do that too, it can just use the alt text we were using previously for the image.

Reply:
further to this, the `track` element require a `src` property to be a url, (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track) so I'll opt out from the rule as the video is `muted` anyway